.github/workflows/checksum-test.yml

```diff
@@ -0,0 +1,29 @@
+name: Checksum Testing
```

Comment:
why is this called `checksum-test.yml`?
It looks like the regular github action for updating `ide-urls`.
Am I missing something?